plugins/config/config.go

```diff
@@ -0,0 +1,30 @@
+package config
+
+import (
+	"fmt"
+	"strings"
+
+	common "github.com/dokku/dokku/plugins/common"
+)
+
+func GetWithDefault(appName string, key string, defaultValue string) string {
```

Comment:
@josegonzalez when I replace this should it fall back on the global config before the default or must this be only app config or default?